src/back/wgsl/writer.rs

```diff
@@ -61,6 +61,7 @@ pub struct Writer<W> {
     names: FastHashMap<NameKey, String>,
     namer: Namer,
     named_expressions: BitSet,
+    ep_results: FastHashMap<ShaderStage, Handle<Type>>,
```

Comment:
how is this supposed to work with multiple entry points? There may be different result types for multiple EPs inside the same module.

Reply:
Thank you! Fixed now.